client/src/components/common/boost/BoostCalibration.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { addArgs, createStory } from 'utils/stories';
+import BoostCalibration from './BoostCalibration';
+
+export default {
+    title: 'BoostCalibration',
```

Comment:
Sort this component with the other boost components as discussed on #team-electrical